src/ILVerify/src/VerifierError.cs

```diff
@@ -146,7 +146,7 @@ enum VerifierError
         //E_ADDR                        "Address of not allowed for this item."
         //E_ADDR_BYREF                  "Address of not allowed for ByRef."
         //E_ADDR_LITERAL                "Address of not allowed for literal field."
-        //E_INITONLY                    "Cannot change initonly field outside its .ctor."
+        Initonly,                       // Cannot change initonly field outside its .ctor.
```

Comment:
@jkotas @ArztSamuel 
Is this file the list of all checks/erros implemented by PEVerify, where all the uncommented ones have been implemented by ILVerify so far?

FYI @VSadov

Reply:
Yes, that is correct.
There are some errors that were added with ILVerify and a lot of commented errors early in the list are metadata related (as opposed to IL related).

Reply:
Is the intention for ILVerify to only verify IL errors, or are metadata errors in scope too? Does it current enforce any metadata rules?

Reply:
ILVerify does basic metadata validation as side-effect of reading the metadata and building the types out of it.

I do not see a reason why we would not want to add more in-depth metadata validation/verification to ILVerify.

Reply:
> I do not see a reason why we would not want to add more in-depth metadata validation/verification to ILVerify.

Yes, I agree. I simply concentrated on IL verifications for now, since basic metadata validation is done as a side-effect anyway, as you mentioned.

Reply:
Awesome. Thanks!